Mathlib/Algebra/Lie/Submodule.lean

```diff
@@ -502,30 +543,46 @@ theorem iSup_induction' {ι} (N : ι → LieSubmodule R L M) {C : (x : M) → (x
   · rintro ⟨_, Cx⟩ ⟨_, Cy⟩
     exact ⟨_, hadd _ _ _ _ Cx Cy⟩
 
-theorem disjoint_iff_coe_toSubmodule :
+theorem disjoint_iff_toSubmodule :
     Disjoint N N' ↔ Disjoint (N : Submodule R M) (N' : Submodule R M) := by
```

Comment:
Uh oh, this lemma is backwards. Not your problem here, but I'd leave a comment to remember it
```suggestion
-- TODO(Yaël): turn around
theorem disjoint_iff_toSubmodule :
    Disjoint N N' ↔ Disjoint (N : Submodule R M) (N' : Submodule R M) := by
```